src/declarativewebcontainer.cpp

```diff
@@ -474,6 +474,7 @@ void DeclarativeWebContainer::clearWindowSurface()
     QOpenGLFunctions_ES2* funcs = m_context->versionFunctions<QOpenGLFunctions_ES2>();
     Q_ASSERT(funcs);
     funcs->glClearColor(1.0, 1.0, 1.0, 0.0);
+    funcs->glScissor(0, 0, width(), height());
```

Comment:
Yep, technically we're clearing the surface, so we should use it's size, not the window. It just happens that in this case the surface is the same size as the window.. Fixed in next version.